JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let hours = Math.floor(seconds / 3600);
+  let mins = Math.floor((seconds % 3600) / 60);
+  let secs = seconds % 60;
+
+  hours < 10 ? (hours = `0${hours}`) : hours;
```

Comment:
please use another variable or function to format the result instead of reassign the value of variable you are using to get hours or mins